src/test/java/org/fundacionjala/coding/rodrigo/CesarCryptoTest.java

```diff
@@ -0,0 +1,28 @@
+package org.fundacionjala.coding.rodrigo;
+
+import org.junit.Test;
+
+/**
+ * Test for the CesarCryptoTest.
+ */
+public class CesarCryptoTest {
+    /**
+     * Test encode.
+     */
+    @Test
+    public void encode() {
+        final int three = 3;
+        final int zero = 3;
+        final int five = 5;
+        CesarCrypto cesarCrypto = new CesarCrypto();
+        System.out.println(cesarCrypto.encode("HOLA", three));
+        System.out.println(cesarCrypto.encode("YA", three));
+        System.out.println(cesarCrypto.encode("HM", three));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)